modules/flox.nix

```
@@ -0,0 +1,39 @@
+pkgsContext:
+{
+  config,
+  pkgs,
+  lib,
+  system,
+  ...
+}:
+
+let
+  cfg = config.programs.flox;
+in
+{
+
+  options = {
+    programs.flox = {
+      enable = lib.mkEnableOption "Flox CLI - Harness the power of Nix";
+      package = lib.mkOption {
+        type = lib.types.package;
+        description = "Flox CLI package";
+        default = pkgsContext.${system}.flox;
```

```suggestion
        default = pkgsContext.${pkgs.system}.flox;
```

This was my mistake. We should use `system` from `pkgs`.